exporters/pelorus/utils/nested.py

```diff
@@ -151,21 +155,21 @@ class BadAttributePathError(Exception):
     root_name is the name of the root item, which makes the error message more helpful.
     """
 
-    root: Any
     path: Sequence[str]
     path_slice: slice
+    key: str
     value: Any
     root_name: Optional[str] = None
 
     @property
     def message(self) -> str:
-        msg = f"{self.root_name + ' is missing' if self.root_name else 'Missing'} {'.'.join(self.path)}"
+        msg = f"{self.root_name + ' is missing' if self.root_name else 'Missing'} {self.key}"
```

Comment:
I changed here because the errors message where not meaningful for me

THEN
```
exporters/tests/test_deserialization.py::test_nested_missing Error while deserializing <class 'tests.test_deserialization.test_nested_missing.<locals>.Nested'> from dict:
  nested_int: Missing foo.bar foo.bar because foo was {}


exporters/tests/test_deserialization.py::test_multi_nested_missing Error while deserializing <class 'tests.test_deserialization.test_multi_nested_missing.<locals>.Nested'> from dict:
  nested_int: Missing foo.com.example.int foo.com.example.int because foo was {}
```
NOW
```
exporters/tests/test_deserialization.py::test_nested_missing Error while deserializing <class 'tests.test_deserialization.test_nested_missing.<locals>.Nested'> from dict:
  nested_int: Missing bar in foo.bar because foo was {}

exporters/tests/test_deserialization.py::test_multi_nested_missing Error while deserializing <class 'tests.test_deserialization.test_multi_nested_missing.<locals>.Nested'> from dict:
  nested_int: Missing com in foo.com[example.int] because foo was {}
```